lib/templates/bootstrap/select.ios.js

```diff
@@ -43,8 +46,8 @@ class CollapsiblePickerIOS extends React.Component {
 
     const height = (this.state.isCollapsed) ? 0 : UIPICKER_HEIGHT;
     return (
-      <View>
-        <TouchableOpacity style={touchableStyle}
+      <View style={[pickerContainer, (!this.state.isCollapsed) ? pickerContainerOpen : {}]}>
```

Comment:
I'm curious, why both `pickerContainer` and `pickerContainerOpen` should be applied as the style of the `View` and why not one or another?

I'm not saying it's not ok, I just want to know the use-case of this style merge.